docs/examples.rst

```diff
@@ -183,42 +271,203 @@ infrastructure defined above, and also define an
+physical quantities we wish to compute, we can submit our desired
+calculations for execution using :py:func:`aospy.submit_mult_calcs`.
+Its sole required argument is a dictionary specifying all of the
+desired parameter combinations.
+
+In the example below, we import and use the ``example_obj_lib`` module
+that is included with aospy and whose objects are essentially
+identical to the ones we've defined above.
+
+.. ipython:: python
+
+    from aospy.examples import example_obj_lib as lib
+
+    calc_suite_specs = dict(
+	library=lib,
+	projects=[lib.example_proj],
+	models=[lib.example_model],
+	runs=[lib.example_run],
+	variables=[lib.precip_largescale, lib.precip_convective,
+                   lib.precip_total, lib.precip_conv_frac],
+	regions='all',
+	date_ranges='default',
+	output_time_intervals=['ann'],
+	output_time_regional_reductions=['av', 'reg.av'],
+	output_vertical_reductions=[None],
+	input_time_intervals=['monthly'],
+	input_time_datatypes=['ts'],
+	input_time_offsets=[None],
+	input_vertical_datatypes=[False],
+    )
+
+:py:func:`submit_mult_calcs` also accepts a second dictionary
+specifying some options regarding how we want aospy to display,
+execute, and save our calculations.  For the sake of this simple
+demonstration, we'll suppress the prompt to confirm the calculations,
+submit them in serial rather than parallel, and suppress writing
+backup output to .tar files:
+
+.. ipython:: python
+
+    calc_exec_options = dict(prompt_verify=False, parallelize=False,
+                             write_to_tar=False)
+
+Now let's submit this for execution:
+
+.. ipython:: python
+
+    from aospy import submit_mult_calcs
+    calcs = submit_mult_calcs(calc_suite_specs, calc_exec_options)
+
+This permutes over all of the parameter settings in
+``calc_suite_specs``, generating and executing the resulting
+calculation.  In this case, it will compute all four variables and
+perform annual averages, both for each gridpoint and regionally
+averaged.
+
+Results
+=======
+
+The result is a list of :py:class:`aospy.Calc` objects, one per
+simulation.
 
 .. ipython:: python
 
-   calc_int = CalcInterface(
-       proj=example_proj,
-       model=example_model,
-       run=example_run,
-       var=precip,
-       date_range=(datetime(4, 1, 1), datetime(6, 12, 31)),
-       intvl_in='monthly',
-       dtype_in_time='ts',
-       intvl_out='ann',
-       dtype_out_time='reg.av',
-       region={'globe': globe}
-   )
-   Calc(calc_int).compute()
+    calcs
+
+Each :py:class:`aospy.Calc` object includes the paths to the output
+and the results of each output type.
+
+.. ipython:: python
+
+    calcs[0].path_out
+    calcs[0].data_out
+
+**(S. Hill: Still need to finish up everything below this point)**
+
+.. note::
+
+   You may have noticed that ``subset_`` and ``raw_`` coordinates have
+   years 1678 and later, when our data was from model years 4
+   through 6.  This is because technical details upstream (in numpy)
```

Comment:
This issue is complicated and there are multiple levels of upstream dependencies; instead of calling out numpy in particular, I might just link to the discussion in this issue: https://github.com/pandas-dev/pandas/issues/7307